blocks/overline-block/index.story.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs";
+
+import Overline from "./features/overline/default";
+
+<Meta title="Blocks/Overline" component={Overline} />
+
+# Byline
+
+The `Overline` component reads from globalContent and returns an overline as a link or span.
+
+## Usage
```

Comment:
We shouldn't document usage examples here as blocks are only to be consumed via PageBuilder and not other blocks/code